media-libs/cimg/cimg-3.1.6.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
Remember to update this Copyright year. There are tools like **pkgdev** when used will do it automatically for you.